docs/01-introduction/03-help.md

```diff
@@ -53,4 +53,4 @@ Since we make use of [Answer Overflow](https://www.answeroverflow.com/c/88308379
 
 ## Helping others
 
-We encourage you to join any of the above platforms and help yourself and our community. Additionally, we encourage you to [contribute](contributing) to Filament itself. We are always looking for new contributors to help us improve Filament.
+We encourage you to join any of the above platforms and help yourself and our community. Additionally, we encourage you to [contribute](05-contributing#development-of-new-features) to Filament itself. We are always looking for new contributors to help us improve Filament.
```

Comment:
Please remove the hash from this link as I appreciate bug fixes as well as new features